### PR TITLE
Missing underscores in docs/guids/helpers.rst

### PR DESCRIPTION
**Pull request by demyxco**
Typos: 

elgg_get_loggedin_user_entity() Returns the ElggUser for the current user
elgg_get_loggedin_user_guid() Returns the GUID of the current user

### CODE REVIEW

**Comment by juho-jaakkola** [comment]
Tests are failing because the commit messages do not conform to the required format: http://learn.elgg.org/en/1.10/contribute/code.html#commit-message-format

(This part is usually a bit tricky for beginners, so let us know if you need help in rewriting the commit messages.)

File: docs/guides/helpers.rst [hunk 1]
```diff
@@ -48,4 +48,4 @@ Interface and annotations
 
 .. code:: php
 
-   elgg_view_entity_icon($user, 'small', array('use_hover' => false));
\ No newline at end of file
+   elgg_view_entity_icon($user, 'small', array('use_hover' => false));
```
Comment (jdalsem) [inline docs/guides/helpers.rst]: what changed on this line?

Comment (manacim) [inline docs/guides/helpers.rst]: -- ``elgg_get_loggedin_user_entity()`` Returns the ElggUser for the current user
-- ``elgg_get_loggedin_user_guid()`` Returns the GUID of the current user
+- ``elgg_get_logged_in_user_entity()`` Returns the ElggUser for the current user
+- ``elgg_get_logged_in_user_guid()`` Returns the GUID of the current user

**Comment by juho-jaakkola** [comment]
Replaced by #8010 and #8011

**Pull request closed without merging**